applications/ls/src/lib.rs

```diff
@@ -34,13 +35,21 @@ pub fn main(args: Vec<String>) -> isize {
         return 0;
     }
 
+    let size_option = matches.opt_present("s");
+
     let Ok(curr_wd) = task::with_current_task(|t| t.get_env().lock().working_dir.clone()) else {
         println!("failed to get current task");
         return -1;
     };
+
+    if size_option && matches.free.is_empty() {
+        print_children(&curr_wd, size_option);
+        return 0;
+    }
```

Comment:
this duplicated code block is unnecessary, you can remove it as it's redundant with the block below it

Reply:
Got it I will remove that